src/proxy.js

```diff
@@ -0,0 +1,79 @@
+/*global require, exports*/
+
+const { rhol, toJSData, fromJSData } = require('./RHOCore');
+const { h2b } = require('./signing');
+const { RNode } = require('./rnodeAPI');
+
+const def = Object.freeze;
+
+/*::
+import type { IRNode } from './rnodeAPI';
+*/
+
+exports.makePeer = makePeer;
+function makePeer(rnode /*: IRNode */, user /*: Uint8Array */,
+                  clock /*: () => Date */,
+                  setTimeout,
+                  // ISSUE: is 1/2 sec between createBlock and listen long enough?
+                  delay=500,
+                  payOpts={ phloLimit: 100000, phloPrice: 1, from: '0x01'}) {
+
+  /**
+   * Note: For better compositionality, JS args are combined into one
+   * list arg on the rholang side.
+   */
```

Comment:
@param target A string representing an RChain registry URI (backticks?)
@param method Name of the method you;d like to call from js
@param args Array or arguments to be passed into rholang contract
@return String of rholang code that executes the requested #transaction